browser/.env.example

```diff
@@ -1,2 +1,3 @@
 VITE_BOOTSTRAP_NODE_LIST="/ip4/127.0.0.1/tcp/10000/ws/p2p/12D3KooWAsxUbeyiTv8zr7iURBh2ugajHK2ZsS8Js4Vu3Q3SrLpA"
-VITE_SIGNAL_SERVER_LIST="sig.encryptafile.com"
\ No newline at end of file
+VITE_SIGNAL_SERVER_LIST="sig.encryptafile.com"
+VITE_DHT_PROTOCOL_PREFIX="archaeologist-server-test"
```

Comment:
If this is set, it should match the value in the matching env var in the service .env file